app/containers/message/index.js

```diff
@@ -80,13 +129,63 @@ export default class MessageContainer extends React.Component {
 	}, 300, true);
 
 	onLongPress = () => {
-		const { archived, onLongPress } = this.props;
+		const {
+			archived, Message_AllowStarring, Message_AllowPinning, Message_Read_Receipt_Store_Users, item, room, user
+		} = this.props;
 		if (this.isInfo || this.hasError || archived) {
 			return;
 		}
-		if (onLongPress) {
-			onLongPress(this.parseMessage());
+		const options = [
```

Comment:
`<Message />` shouldn't be as leaner as possible.
This kind of methods needs to be on the parent (RoomView).
Then we'll have one method on the parent pointing to all children instead of the method being created in every `<Message />` in the list.